client/src/extension.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  TransportKind,
+} from 'vscode-languageclient'
+import { ExtensionContext, workspace } from 'vscode'
+import * as path from 'path'
```

Comment:
I would personally turn on `esModuleInterop: true` in the tsconfig.json, then you can just do `import path from 'path'`.